README.md

```diff
@@ -1 +1,106 @@
+<a name="readme-top"></a>
 
+
+<div align="center">
+  
+
+  <h3><b>Hello Microverse!</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [Hello Microverse!] <a name="about-project"></a>
+
+**[Hello Microverse]** is The Day 2 of the Microverse Program which includes creating linters, branching, monitoring github workflow and editing the readme file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running follow these simple steps:
+
+### Prerequisites
+
+- Git Installed
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open a terminal
+- Run ```git clone git@github.com:jerymey/Hello-Microverse.git```
+- run ``` cd Hello-Microverse/views```
+- open index.html
+
+### Install
+
+Install this project with:
+
+cd Hello-microverse
+  gem install
```

Comment:
- 🙏Please remove this instruction since it is not valid.This instruction is valid for a Ruby💎 project but is not valid for this simple HTML project